plugin/adsk/scripts/USDMenuProc.mel

```diff
@@ -55,6 +55,21 @@ global proc mayaUsdMenu_duplicate(string $ufePath)
     }
 }
 
+global proc mayaUsdMenu_unassignMaterial(string $ufePath)
+{
+    if (!hasPrimUpdater())
```

Comment:
I'm not 100% sure you need this check. This was to protect against editing-as-Maya non-editable objects. Not sure it applies to assigning materials.